CONTRIBUTING.md

```diff
@@ -7,10 +7,8 @@ merged in.  If you do not wish for ChaosBot to consider your PR for merging just
 yet, add "WIP" somewhere in your PR title.  Remove it when you're ready for voting.
 
 # Changing your PR
-It is very important that you avoid updating your PR with new commits after it
-has been created.  The reason for this is that ChaosBot will only consider votes
-that were cast **after** the last code change timestamp.  So if you collect a
-bunch of votes and PR, you have lost those votes.
+You may change your PR at any time without losing votes, but keep in mind, any
+new changes will reset the vote window for additional time.
```

Comment:
resets the starting point from which marks the beginning of the voting time.  appears to be an extension though